Client/Frontend/Settings/ThemeSettings/ThemeSettingsState.swift

```diff
@@ -12,6 +12,10 @@ struct ThemeSettingsState: ScreenState, Equatable {
     var userBrightnessThreshold: Float
     private var logger: Logger
 
+    var systemBrightness: Float {
```

Comment:
@OrlaM I've added `systemBrightness` as a computed var because is not directly related to the ThemeManager so I wasn't sure if it should go through the middleware or not. What do you think?